pkg/flare/flare.go

```diff
@@ -38,8 +38,8 @@ func SendFlareWithHostname(archivePath string, caseID string, email string, host
 	//Write stuff to the pipe will block until it is read from the other end, so we don't load everything in memory
 	go func() {
 
-		defer bodyWriter.Close()
 		defer writer.Close()
+		defer bodyWriter.Close()
```

Comment:
The defers execute in inverse order as they are written (they are "stacked"), so I think the previous order is the correct one (although probably doesn't matter much).

Reply:
You are right, and it appears that it matters (unit tests broke when I changed).
Thanks!